src/components/elements/input/index.tsx

```diff
@@ -1,17 +1,21 @@
 import { forwardRef } from 'react';
 
-import { StyledInput, StyledInputBorder, StyledInputWrapper } from './input.styles';
+import { StyledInput, StyledInputBorder, StyledInputWrapper, StyledSearchButton } from './input.styles';
 import type { InputProps } from './input.types';
 import { MagnifyingGlass } from '../icon';
 import { Text } from '../text';
 
 export const Input = forwardRef<HTMLInputElement, InputProps>(
```

Comment:
@RBoraurito, it seems this is not an element anymore since we are using two other elements to build it(Icon/Text). Can we move it to the modules folder?

Reply:
Of course, we can, would be more accurate now that I think about it.